### PR TITLE
i18n: Add Slovenian translation

### PR DESCRIPTION
**Pull request by matevz**
Adds Slovenian translation of the Oasis Wallet.

image
image
image
image

### CODE REVIEW

**Review by tjanez — COMMENTED** [review]
Can you squash all changes into a single commit?

File: src/locales/sl/translation.json [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    "subnavigation": {
+      "transactions": "Transakcije",
+      "activeDelegations_0": "Aktivne delegacije ({{count}})",
+      "activeDelegations_1": "Aktivne delegacije ({{count}})",
+      "activeDelegations_2": "Aktivne delegacije ({{count}})",
+      "activeDelegations_3": "Aktivne delegacije ({{count}})",
+      "debondingDelegations_0": "Delegacije v razvezi ({{count}})",
+      "debondingDelegations_1": "Delegacije v razvezi ({{count}})",
+      "debondingDelegations_2": "Delegacije v razvezi ({{count}})",
+      "debondingDelegations_3": "Delegacije v razvezi ({{count}})"
```
Comment (lukaw3d) [inline src/locales/sl/translation.json:105]: Was this for testing?

English version: https://github.com/oasisprotocol/oasis-wallet-web/blob/1b059720f94aad0c3bec69599b1d19e03543f3b4/src/locales/en/translation.json#L103-L106

Comment (matevz) [reply src/locales/sl/translation.json:105]: No, this is correct. Slovenian has four types of plurals. English only has two.

For this particular case I decided it's better to always show the plural term, so the menu entry remains intact when you switch the wallet/network. The number in parenthesis changes and that's clear enough IMO.

Comment (lukaw3d) [reply src/locales/sl/translation.json:105]: Oh, this is weird
```
-1 => activeDelegations_1
 0 => activeDelegations_0
 1 => activeDelegations_1
 2 => activeDelegations_2
 3 => activeDelegations_3
 4 => activeDelegations_3
 5 => activeDelegations_0
 6 => activeDelegations_0
```

Comment (matevz) [reply src/locales/sl/translation.json:105]: Is this taken from i18next? It seems correct. Take a female noun, first declension:
```
-1 stopinja
0 stopinj
1 stopinja
2 stopinji
3 stopinje
4 stopinje
5 stopinj
6 stopinj
```

**Review by tjanez — APPROVED** [review]

**Pull request merged**